policyengine_us/parameters/gov/states/ar/tax/income/deductions/standard.yaml

```
@@ -0,0 +1,23 @@
+description: Arkansas provides a standard deduction of this amount.
+metadata:
+  label: Arkansas standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Code of Arkansas Public Access A.C.A. Subchapter 4. § 26-51-430  
+    href: https://advance.lexis.com/container/?pdmfid=1000516&crid=1b7c59fc-ee50-4b71-9f32-869742d3b94c&func=LN.Advance.ContentView.getFullToc&nodeid=ABAAAFAAC&typeofentry=Breadcrumb&config=00JAA3ZTU0NTIzYy0zZDEyLTRhYmQtYmRmMS1iMWIxNDgxYWMxZTQKAFBvZENhdGFsb2cubRW4ifTiwi5vLw6cI1uX&action=publictoc&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A649P-FBD0-R03K-54BT-00008-00&pdtocfullpath=%2Fshared%2Ftableofcontents%2Furn%3AcontentItem%3A50XD-80G1-DY3X-63F1-00008-00&ecomp=-3vckkk&prid=e3cf7c68-667d-45c5-b52a-91f7e551c867
```

Direct link to section
```suggestion
    href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=8a0a4a46-62e0-41b6-8539-42452c3a9e46&nodeid=ABAAAFAACAAFABG&nodepath=%2FROOT%2FABA%2FABAAAF%2FABAAAFAAC%2FABAAAFAACAAF%2FABAAAFAACAAFABG&level=5&haschildren=&populated=false&title=26-51-430.+Deductions+%E2%80%94+Standard+deduction+%E2%80%94+Definition.&config=00JAA2ZjZiM2VhNS0wNTVlLTQ3NzUtYjQzYy0yYWZmODJiODRmMDYKAFBvZENhdGFsb2fXiYCnsel0plIgqpYkw9PK&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A649P-FBD0-R03K-54BT-00008-00&ecomp=7gf5kkk&prid=4b6d04eb-5f42-47a5-b744-f8596299a894
```